doc/src/fix_wall_gran.rst

```diff
@@ -8,7 +8,7 @@ Syntax
 
 .. parsed-literal::
 
-   fix ID group-ID wall/gran fstyle fstyle_params wallstyle args keyword values ...
+   fix ID group-ID wall/gran fstyle fstyle_params wallstyle args keyword values ... [store_contacts]
```

Comment:
not clear why store_contacts is listed here - it's just one of several possible keywords - also we don't typically put underscores in keywords, just 
"contacts" would be better